### ISSUE TITLE
Security vulnerabilities report

### ISSUE THREAD

**Issue by TheoBrigitte** [getsentry/sentry-go#465]
## Summary

On latest [`master` ](https://github.com/getsentry/sentry-go/commit/26ea60338007cb88445870a06353fb79df9d8338), [nancy](https://github.com/sonatype-nexus-community/nancy) found 4 vulnerable dependencies with 5 high to critical CVEs.

- pkg:golang/github.com/kataras/iris/v12@v12.1.8 : [CVE-2021-23772](https://ossindex.sonatype.org/vulnerability/CVE-2021-23772?component-type=golang&component-name=github.com%2Fkataras%2Firis%2Fv12&utm_source=nancy-client&utm_medium=integration&utm_content=0.0.0-dev) ( CVSS Score : 8.8/10 (High) )
- pkg:golang/github.com/microcosm-cc/bluemonday@v1.0.2 : [CVE-2021-42576](https://ossindex.sonatype.org/vulnerability/CVE-2021-42576?component-type=golang&component-name=github.com%2Fmicrocosm-cc%2Fbluemonday&utm_source=nancy-client&utm_medium=integration&utm_content=0.0.0-dev) ( CVSS Score : 9.8/10 (Critical )
- pkg:golang/github.com/nats-io/jwt@v0.3.0 : [CVE-2020-26892](https://ossindex.sonatype.org/vulnerability/CVE-2020-26892?component-type=golang&component-name=github.com%2Fnats-io%2Fjwt&utm_source=nancy-client&utm_medium=integration&utm_content=0.0.0-dev) ( CVSS Score : 9.8/10 (Critical) )
- pkg:golang/github.com/nats-io/jwt@v0.3.0 : [CVE-2021-3127](https://ossindex.sonatype.org/vulnerability/CVE-2021-3127?component-type=golang&component-name=github.com%2Fnats-io%2Fjwt&utm_source=nancy-client&utm_medium=integration&utm_content=0.0.0-dev) ( CVSS Score : 7.5/10 (High) )
- pkg:golang/github.com/valyala/fasthttp@v1.6.0 : [CVE-2022-21221](https://ossindex.sonatype.org/vulnerability/CVE-2022-21221?component-type=golang&component-name=github.com%2Fvalyala%2Ffasthttp&utm_source=nancy-client&utm_medium=integration&utm_content=0.0.0-dev) ( CVSS Score : 7.5/10 (High) )

## Steps To Reproduce

```bash
$ git checkout master
$ CGO_ENABLED=0 go list -json -m all | nancy sleuth --skip-update-check --quiet -x /dev/null
...

5 Vulnerable Packages

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Summary                       ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━┫
┃ Audited Dependencies    ┃ 129 ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━┫
┃ Vulnerable Dependencies ┃ 5   ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━┛
```

## Expected Behavior

Zero security vulnerability found.

```
$ CGO_ENABLED=0 go list -json -m all | nancy sleuth --skip-update-check --quiet -x /dev/null
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Summary                       ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━┫
┃ Audited Dependencies    ┃ 136 ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━┫
┃ Vulnerable Dependencies ┃ 0   ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━┛
```

## Environment

```
- Go version: 1.19
- Nancy version: 1.0.37-1
- sentry-go version: 26ea60338007cb88445870a06353fb79df9d8338
```

## Additional context

This was already reported in multiple issues: #423 #438 #445

**Comment by mmgfrcs** [comment]
Any news on this?

**Comment by pbennett** [comment]
There are significant issues in the CAR dependency as well:
https://github.com/advisories/GHSA-9x4h-8wgm-8xfg
github.com/ipld/go-car needs upgraded to at least 0.4.0

**Comment by TheoBrigitte** [comment]
This was merged into master (commit [33f7b69](https://github.com/getsentry/sentry-go/commit/33f7b6930327848cea5fdb2bdc65b5992632aef9))
